src/extensions/service-vendors/bandwidth/messaging.js

```diff
@@ -178,19 +178,38 @@ export async function postMessageSend({
         ...changes
       }
     : {};
+  const organizationContact = {
+    contact_number: message.contact_number,
+    organization_id: organization.id,
+    service: "bandwidth"
+  };
   if (response && response.statusCode === 202 && response.result) {
     changesToSave.service_id = response.result.id;
+    organizationContact.status_code = 1;
+    organizationContact.user_number = response.result.from;
+    cacheableData.campaignContact.updateStatus(
+      contact,
+      undefined,
+      changesToSave.messageservice_sid || changesToSave.user_number
+    );
   } else {
     // ERROR
     changesToSave.send_status = "ERROR";
     // TODO: maybe there is sometimes an error response in the JSON?
```

Comment:
would this todo be resolved by your change on line 200? If so, can we delete this todo to avoid confusion?